Header header
int32 numeroCaras
int32[] x1
int32[] y1
int32[] x2
int32[] y2
int32[] puntoX
int32[] puntoY
int32[] indiceCara
bool[] caraNueva

